scss/mixins/_forms.scss

```diff
@@ -123,5 +123,12 @@
       margin-left: .5em;
     }
   }
+
+  .input-group .form-control,
+  .input-group .form-select {
```

Comment:
Can this be moved to `forms/_input-group.scss`?

Reply:
No, this rely on `$state` to work. The issue you opened is only another edge case/context for validated fields (in input group),  not an input group feature.

Reply:
:+1: